ckan/lib/jobs.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+u'''
+Asynchronous background jobs.
+
+Note that most job management functions are not available from this
+module but via the various ``job_*`` API functions.
+
+Internally, RQ queue names are prefixed with a string containing the
+CKAN site ID to avoid key collisions when the same Redis database is
+used for multiple CKAN instances. The functions of this module expect
+unprefixed queue names (e.g. ``'default'``) unless noted otherwise. The
+raw RQ objects (e.g. a queue returned by ``get_queue``) use the full,
+prefixed names. Use the functions ``add_queue_name_prefix`` and
+``remove_queue_name_prefix`` to manage queue name prefixes.
+
+.. versionadded:: 2.6
+'''
+
+import logging
+
+from pylons import config
```

Comment:
Since #3163 was merged we should never import `config` directly from Pylons anymore. Use `from ckan.common import config`.

My fault for not documenting / announcing it